test/gtest/ucp/test_ucp_am.cc

```diff
@@ -1244,11 +1255,6 @@ UCS_TEST_P(test_ucp_am_nbx_dts, long_zcopy_send, "ZCOPY_THRESH=1",
     test_am(64 * UCS_KBYTE);
 }
 
-UCS_TEST_P(test_ucp_am_nbx_dts, send_eager_flag, "RNDV_THRESH=128")
```

Comment:
maybe reduce number of tests here even more? (as they are run for all combinations of dts and flags). Say we can test just `bcopy` and `zcopy` sends. Or maybe do just one test which sends messages of several  random sizes

Reply:
at least, we can unite 4K and 64K in single test instance
anyway I'd like to merge this PR since it helps other PRs to pass test faster, and we can optimize it more in next PR